internal/commands/redteam/redteam.go

```diff
@@ -179,9 +191,20 @@ or use the --config flag to specify a custom path.`
 		},
 		Options: redteamclient.AIScanOptions{
 			VulnDefinitions: redTeamConfig.Options.VulnDefinitions,
+			ScanningAgent:   redTeamConfig.Options.ScanningAgent,
 		},
 	}
 
+	scanningAgentIDOverride := config.GetString(utils.FlagRedTeamScanningAgentID)
+	if scanningAgentIDOverride != "" {
+		validate := validator.New()
+		clientConfigErr := validate.Var(scanningAgentIDOverride, "uuid")
+		if clientConfigErr != nil {
+			return nil, nil, cli_errors.NewValidationFailureError(clientConfigErr.Error())
```

Comment:
how does this error look like to the user? Might not be really friendly I would customise it in this case.

Reply:
What happens now? Is the error bad request and the message you change wrapped within?